### PR TITLE
Minor Readme cosmetics

### PR DESCRIPTION
**Pull request by rasbt**
I was just reading through the README and noticed a few points and thought making a quick PR doesn't hurt :)

### CODE REVIEW

**Review by t-vi — APPROVED** [review]
Thank you @rasbt

File: README.md [hunk 1]
```diff
@@ -119,11 +119,11 @@ print(result)
 
 The compiled function `jfoo` takes and returns PyTorch tensors, just like the original function, so modules and functions compiled by Thunder can be used as part of larger PyTorch programs.
 
-## Train models
+## Model training with Thunder
```
Comment (lantiga) [inline README.md:122]: Let's keep `Train models` as the title

**Review by lantiga — APPROVED** [review]
Just one change

File: README.md (hunk 1 above)
Comment (rasbt) [reply README.md:122]: I don't know where the "Train models" came from. In my copy it said "Running training" hence the change. But I like "Train models" even better than "Model training with models"

**Review by Borda — APPROVED** [review]

**Pull request merged**